cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -205,6 +205,16 @@ public static Kafka createKafka(String namespace, String name, int replicas,
                 .build();
     }
 
+    public static Secret createSecret(String name, String namespace, Map<String, String> data) {
```

Comment:
Don't we have this already somewhere? What about AbstractModel or something like that?